client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
In other words, this PR splits our first move towards a single page approach back into a multiple page approach. I understand that this is temporary but I wonder if it is possible to continue/reinvent the single page approach within this PR. It would be great to navigate through Galaxy without page reloads and tours would work great too.